runtime/gc_include/ObjectAccessBarrierAPI.hpp

```diff
@@ -2789,27 +2789,33 @@ class MM_ObjectAccessBarrierAPI
 	/**
 	 * Check to see if the object is marked
 	 *
-	 * @NOTE only for WRT as this check the mark map for segregated heaps
+	 * @NOTE only for SATB Concurrent Mark
 	 *
 	 * @param object to check
 	 * @return true if the object was marked, false otherwise
 	 */
 	VMINLINE bool
 	isMarked(J9VMThread *vmThread, j9object_t object)
 	{
-		UDATA heapMapBits = 0;
 		J9JavaVM *const javaVM = vmThread->javaVM;
-		UDATA heapDelta = (UDATA)object - javaVM->realtimeHeapMapBasePageRounded;
-		UDATA slotIndex = heapDelta >> J9VMGC_SIZECLASSES_LOG_SMALLEST;
- 		UDATA bitIndex = slotIndex & J9_GC_MARK_MAP_UDATA_MASK;
-		UDATA bitMask = ((UDATA)1) << bitIndex;
-		slotIndex = slotIndex >> J9_GC_MARK_MAP_LOG_SIZEOF_UDATA;
 
-		UDATA *heapMapBitsBase = javaVM->realtimeHeapMapBits;
-		heapMapBits = heapMapBitsBase[slotIndex];
-		heapMapBits &= bitMask;
+		if (J9_GC_POLICY_METRONOME == javaVM->gcPolicy) {
+			UDATA heapMapBits = 0;
+			UDATA heapDelta = (UDATA)object - javaVM->realtimeHeapMapBasePageRounded;
+			UDATA slotIndex = heapDelta >> J9VMGC_SIZECLASSES_LOG_SMALLEST;
+			UDATA bitIndex = slotIndex & J9_GC_MARK_MAP_UDATA_MASK;
+			UDATA bitMask = ((UDATA)1) << bitIndex;
+			slotIndex = slotIndex >> J9_GC_MARK_MAP_LOG_SIZEOF_UDATA;
 
-		return (0 != heapMapBits);
+			UDATA *heapMapBitsBase = javaVM->realtimeHeapMapBits;
+			heapMapBits = heapMapBitsBase[slotIndex];
+			heapMapBits &= bitMask;
+
+			return 0 != heapMapBits;
```

Comment:
restore the oval brackets

Reply:
That is contrary to the coding standard (openj9 generally follow the OMR guidance); see https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md which says explicitly:
> Do not enclose the return value in parentheses.

Reply:
ok, let's follow the coding standard, then
btw, there are many (possibly majority) spots that do enclose